.travis.yml

```diff
@@ -18,8 +18,10 @@ before_install:
 - echo HRNGDEVICE=/dev/urandom | sudo tee /etc/default/rng-tools
 - sudo /etc/init.d/rng-tools restart
 install:
-- pip install ansible==1.8.4
-- pip install coveralls
+- pip freeze -l
+# Preinstalled travis pytest is not tracked by us
+- pip uninstall pytest -y
+- pip install -r testinfra/requirements.txt
```

Comment:
The issue with your suggestion is that I moved the `ansible` requirement into the `testinfra/requirements.txt` and you need ansible before you can install the `test-requirements.txt`.

I guess we could go back to a single travis-requirements.txt file that just tracks the ansible version and then everything else could be shoved in `test-requirements.txt` ? Would that be a good compromise? The reason I took `ansible` out of the `travis.yml` because I thought it was better to track all the pip dependencies in dedicated requirements files - its more obvious where to look to when you need to find them